buildtest/cli/__init__.py

```diff
@@ -396,6 +398,23 @@ def misc_menu(subparsers):
         ],
         help="Show help message for command",
     )
+def profile_menu(subparsers):
+
+    config_profiles = subparsers_buildpsec.add_parser(
+        parents=[parent_parser["theme"]],
+    )
+    parent_parser["theme"] = argparse.ArgumentParser(add_help=False)
+    parent_parser["theme"].add_argument(
+        "-t",
+        compliers="buildtest config profile",
```

Comment:
why do you have this line, please remove `compliers="buildtest config profile",`